src/containers/Dashboard/Dashboard.module.scss

```diff
@@ -18,52 +20,107 @@
 }
 
 .mobileContainer {
-  display: none;
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  overflow-x: hidden;
+  min-height: $slider-height;
 
-  @media only screen and (max-width: 48em) {
-    display: block;
+  @media only screen and (min-width: 48em) {
+    display: none;
+  }
+}
+
+.panelContainer {
+  display: flex;
+  flex-wrap: wrap;
+  min-height: $slider-height;
+  height: 100%;
+}
+
+.bountiesPanel,
+.activityPanel,
+.submissionsPanel {
+  box-shadow: none;
+  border-radius: 0;
+
+  @media only screen and (min-width: 48em) {
+    box-shadow: $base-box-shadow;
+    border-radius: $base-border-radius;
   }
 }
 
 .bountiesPanel {
+  min-height: $slider-height;
+  height: 100%;
+  width: 100%;
+
   @media only screen and (min-width: 48em) {
     display: flex;
     flex-direction: column;
     flex-grow: 1;
-    height: 27vh;
-    min-height: 25rem;
+    min-height: 0;
+    height: 30rem;
```

Comment:
should we use a variable here?

Reply:
Good catch, I'll go in and replace that.